pangeo_forge_recipes/transforms.py

```diff
@@ -56,7 +57,7 @@
 
 
 # TODO: replace with beam.MapTuple?
-def _add_keys(func):
+def _add_keys(func: Callable) -> Callable:
```

Comment:
A Callable of what?

Reply:
Touché, @alxmrs, that was a cunning nerd snipe. 😆 

To find the answer, I just went down the (interesting, not-too-deep) rabbit hole of [PEP612](https://peps.python.org/pep-0612/), and arrived at https://github.com/pangeo-forge/pangeo-forge-recipes/pull/557/commits/5ebb0f72518eb6bd4e4984c5f40a9fe12f0fea27.

Do those hints look plausible/intelligible to you?

Reply:
I feel like these annotation are correct! To keep things moving, I'm going to merge, @alxmrs if you catch anything out-of-place, feel free to let me know and I'll fix it in a follow-on.

Thanks for encouraging more specificity here, I think this really helps with clarity. 😃